lesson01.calculator/index.html

```diff
@@ -1,22 +1,294 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <title>Web Calculator</title>
     <meta charset="utf-8">
-    <meta name="viewport"
-          content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <style>
-        body {
+        h1 {
+            width: 300px;
+            margin: 0 auto;
+            text-align: center;
+        }
+
+        main {
+            height: 400px;
+            width: 500px;
+            background-color: white;
+            margin: 0 auto;
+        }
+
+        #calc {
+            border: 1px solid black;
+            padding: 15px;
+            width: 275px;
+            height: 275px;
+            margin: 0 auto;
+        }
+
+        #calcRe {
+            /* border: 1px solid black; */
+            background-color: white;
+            width: 250px;
+            height: 25px;
+            font-size: 15px;
+            text-align: left;
+        }
+
+        #calcInput {
+            border: 1px solid black;
+            background-color: white;
+            width: 250px;
+            height: 30px;
+            font-size: 25px;
+        }
+
+        table,
+        th,
+        td {
+            font-family: Arial, sans-serif;
+        }
+
+        input {
+            height: 50px;
+            width: 50px;
+        }
+
+        input#num0 {
+            width: 105px;
+        }
 
+        input#opEql,
+        input#opClr {
+            height: 105px;
         }
     </style>
 </head>
+
 <body>
     <header>
         <h1>Web Calculator</h1>
     </header>
     <main>
-        Content Here
+        <div id="calc">
+            <table>
+                <thead>
+                    <tr>
+                        <th colspan="5" id="calcRe"></th>
+                    </tr>
+                    <tr>
+                        <th colspan="5" id="calcInput">0</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td><input type="button" id="num7" value="7" onclick="input(this.value)"></td>
```

Comment:
`input` is not a good name for a function. name it after what the function does (treatNumber, add, subtract) etc.